napari/_vispy/layers/base.py

```diff
@@ -230,33 +231,22 @@ def _on_matrix_change(self):
             affine_offset = np.eye(4)
             affine_offset[-1, : len(offset)] = offset[::-1]
             affine_matrix = affine_matrix @ affine_offset
+            if self.layer.multiscale:
+                # Because of performance reason, for multiscale images
+                # we load only visible part of data to GPU.
+                # To place this part of data correctly we update transform,
+                # but this leads to incorrect placement of child layers.
+                # To fix this we need to update child layers transform.
```

Comment:
```suggestion
                # Because of performance reason, for multiscale images
                # we load only visible part of data to GPU.
                # To place this part of data correctly we update transform,
                # but this leads to incorrect placement of child nodes (e.g overlays).
                # To fix this we need to update child layers transform.
```